arbitrary-users-patch/base_images

```diff
@@ -1,7 +1,7 @@
 che-cpp-rhel7           registry.access.redhat.com/devtools/llvm-toolset-rhel7
 che-dotnet-2.2          mcr.microsoft.com/dotnet/core/sdk:2.2-stretch
 che-golang-1.12         golang:1.12-stretch
-che-java11-gradle       gradle:5.2.1-jdk11
+che-java11-gradle       gradle:6.2.1-jdk11
```

Comment:
The corresponding meta.yaml for gradle should be updated, since it still refers to 5.2.1 in description.

Reply:
fixed